leku/src/main/java/com/schibstedspain/leku/CountryLocaleRect.java

```diff
@@ -55,11 +55,11 @@ static LatLng getLowerLeftFromZone(Locale locale) {
       return GERMANY_LOWER_LEFT;
     } else if (Locale.GERMAN.equals(locale)) {
       return GERMAN_LOWER_LEFT;
-    } else if (UAE_COUNTRY_CODE.equals(locale.getCountry())) {
+    } else if (locale.toString().toLowerCase().contains(UAE_COUNTRY_CODE)) {
```

Comment:
Can this type of check lead to false positives? Maybe we should split string by lowercase and get the second part to check it?

Reply:
It could happen but there are so few that could collide. But yes, we must use the full lowercase locale